igvc_navigation/launch/pather.launch

```diff
@@ -18,16 +18,17 @@
         <param name="theta_filter" type="double" value="1.5"/>
         <param name="max_theta_change" type="double" value="0.8"/>
         <param name="theta_change_window" type="double" value="3.0"/>
+        <param name="maximum_distance" type="double" value="100.0"/>
     </node>
 
     <!-- Follower -->
     <node name="path_follower" pkg="igvc_navigation" type="path_follower" output="screen" >
         <param name="maximum_vel" type="double" value="6.0"/>
         <param name="target_v" type="double" value="0.6"/>
         <param name="axle_length" type="double" value="0.52"/>
-        <param name="lookahead_dist" type="double" value="1.8"/>
-        <param name="k1" type="double" value="0.5"/>
-        <param name="k2" type="double" value="1.0"/>
+        <param name="lookahead_dist" type="double" value="1.3"/>
```

Comment:
might have to retune on the actual robot

Reply:
this is an important thing we have to consider. The paper doesn't really do the best job explaining how these parameters affect the trajectory but, from observing the robot's behavior with different values of k1 and k2 I noticed that k1 affects how theta changes relative to the distance from the target pose and k2 affects how quickly the robot converges to the target pose.

I found these values of k1 and k2 to do a pretty good job at getting the robot on the path